modules/monitoring-creating-cluster-monitoring-configmap.adoc

```diff
@@ -3,36 +3,55 @@
 // * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
 
 [id="creating-cluster-monitoring-configmap_{context}"]
-= Creating cluster monitoring ConfigMap
+= Creating a cluster monitoring ConfigMap
 
-To configure the Prometheus Cluster Monitoring stack, you must create the cluster monitoring ConfigMap.
+To configure the {product-title} monitoring stack, you must create a cluster monitoring ConfigMap.
```

Comment:
ConfigMap here to be created has specific name and namespace, suppose "the" is better, should not user "a"